src/wake/WakeCategoryEventHandler.cpp

```diff
@@ -45,30 +48,151 @@ namespace UKControllerPlugin {
 
         std::string WakeCategoryEventHandler::GetTagItemDescription(int tagItemId) const
         {
-            return "Aircraft Type / UK Wake Category";
+            if (tagItemId == this->tagItemIdAircraftTypeCategory) {

```

Comment:
I might, for once, suggest the use of a switch statement to reduce the number of return statements.

Reply:
Annoyingly I can't :( Unless I start using `#define` in the code which isn't great - anything in a switch has to be compile time constant :(

Reply:
Made it all modern like!

Reply:
I might steal this :D nice work!